jsapp/js/components/modalForms/submission.es6

```diff
@@ -246,14 +246,15 @@ class Submission extends React.Component {
       submissionValue = overrideValue;
 
     switch(q.type) {
-      case 'select_one':
+      case 'select_one': {
         const choice = choices.find(x => x.list_name == q.select_from_list_name && x.name === submissionValue);
         if (choice && choice.label && choice.label[translationIndex])
           return choice.label[translationIndex];
         else
           return submissionValue;
         break;
-      case 'select_multiple':
+      }
+      case 'select_multiple': {
         var responses = submissionValue.split(' ');
         var list = responses.map((r)=> {
```

Comment:
According to the rules, you've described? Should this one be `const list` instead `var list` too?